CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Changed
+* Mandate ispe and disallow zero width or height (#640).
```

Comment:
Joe: This change is part of your "progressive" branch but it is independent of progressive decoding, so I wanted to make sure we describe it separately in the CHANGELOG.

Could you also check the "Re-map libavif speed 7-10 to libaom speed 7-9 (#682)}" item at lines 11-16? This item has multiple lines. I am not sure if we wrote it correctly.